library/network/src/lib/y2firewall/firewalld/api/services.rb

```diff
@@ -44,8 +44,12 @@ def services
         # @param permanent [Boolean] if true it adds the --permanent option the
         # command to be executed
         # @return [Array<String>] list of all information for the given service
-        def info_service(service, permanent: permanent?)
-          string_command("--info-service", service.to_s, permanent: permanent).split("\n")
+        def info_service(service, permanent: permanent?, verbose: false)
+          if verbose
+            string_command("--info-service=#{service}", "--verbose", permanent: permanent).split("\n")
```

Comment:
What about building an array (and inserting the `--verbose` string if needed) and having only one call to `string_command`?

Reply:
Yep, I already was thinking on that